index.css

```diff
@@ -0,0 +1,110 @@
+{box-sizing: border-box}
+
+
+
+body{
+
+}
+
+nav{
+    height: 120px;
+    width: 100%;
+    position: fixed;
+    top: 0;
+    z-index: 3;
+}
+
+nav img{
+    margin-left: 50px;
+}
+
+.navRight{
+    float: right;
+    margin-right: 200px;
+    padding-top: 30px;
+}
+
+
+.dropdown{
+    float: right;
+    margin-right: 200px;
+    margin-top: 70px;
+    display: none;
+    background-image: url("button.png");
+    background-size: 100%;
+    height: 5px;
+    width: 5px;
+    padding: 16px;
+    border: none;
+}
+
+nav li{
+    display: inline-block;
+    margin-top: 30px;
+}
+
+.liText{
```

Comment:
not a great class name
think about what it actually is eg: navItem

Reply:
actually to be honest you dont need a class. just use a css selector like 
.navRight li a {
//styles
}